config/helm/appmesh-controller/values.yaml

```diff
@@ -43,7 +43,7 @@ init:
 
 xray:
   image:
-    repository: amazon/aws-xray-daemon
+    repository: public.ecr.aws/xray/aws-xray-daemon
     tag: latest
```

Comment:
Can we change latest to the tag value instead?

Reply:
The latest docker image tag is still available on ECR public repo of aws-xray-daemon. Everytime they upload a new version they would update their latest I believe. So it is good to still keep this default tag to `latest` still?

Reply:
The reason of being specific is that, if a someone runs into an issue, we can quickly narrow it down.

Reply:
As aws-xray-daemon is been stable for all these time, I think we can still keep the latest tag.